src/utils/strings/frenchConstants.tsx

```diff
@@ -0,0 +1,918 @@
+import { Box, Typography } from '@mui/material';
+import Link from '@mui/material/Link';
+import LinkButton from 'components/pages/gallery/LinkButton';
+import React from 'react';
+import { SuggestionType } from 'types/search';
+import { formatNumberWithCommas } from '.';
+
+/**
+ * Global French constants.
+ */
+
+const dateString = function (date) {
+    return new Date(date / 1000).toLocaleDateString('fr-FR', {
+        year: 'numeric',
+        month: 'long',
+        day: 'numeric',
+    });
+};
+
+const frenchConstants = {
+    HERO_SLIDE_1_TITLE: () => (
+        <>
+            <div>Private backups</div>
+            <div> for your memories</div>
+        </>
+    ),
+    HERO_SLIDE_1: 'Chiffrement de bout en bout par défaut',
+    HERO_SLIDE_2_TITLE: () => (
+        <>
+            <div>Safely stored </div>
+            <div>at a fallout shelter</div>
+        </>
+    ),
+    HERO_SLIDE_2: 'Conçu pour survivre',
+    HERO_SLIDE_3_TITLE: () => (
+        <>
+            <div>Available</div>
+            <div> everywhere</div>
+        </>
+    ),
+    HERO_SLIDE_3: 'Android, iOS, Web, Ordinateur',
+    COMPANY_NAME: 'ente',
+    LOGIN: 'Connexion',
+    SIGN_UP: 'Inscription',
+    NEW_USER: 'Nouveau sur ente',
+    EXISTING_USER: 'Utilisateur existant',
+    NAME: 'Nom',
+    ENTER_NAME: 'Saisir un nom',
+    PUBLIC_UPLOADER_NAME_MESSAGE:
+        'Ajouter un nom afin que vos amis sachent qui remercier pour ces magnifiques photos!',
+    EMAIL: 'E-mail',
+    ENTER_EMAIL: 'Saisir l''adresse e-mail',
```

Comment:
there are two single quotes after l, `l''adresse`
is it expected?
or should it have been `l'adresse` ?

Reply:
Ahh okay, understood!
We are handling it a bit differently, I will update the code according.